app/views/category/index.html.erb

```diff
@@ -0,0 +1,153 @@
+	<!DOCTYPE html>
+<html lang="pt">
+<style>
+body{
```

Comment:
Esse estilo do `body` não é o mesmo que está no `application.scss`? Se for, não é necessário por aqui.